Add guide for sigs (#60)
hyphacoop/handbook: guides.md

As a last resort for sensitive docs, a member can choose to have the document **printed and stored** in a folder in the office.

## Signatures

> 

Reviewer: ```suggestion

### Signing internal documents

```

The Board of Directors uses PGP signatures to digitally sign [resolutions](https://github.com/hyphacoop/organizing/tree/master/resolutions).

> We recommend using the [GnuPG](https://gnupg.org) command line, but you can also use the [Keybase](https://keybase.io) command line, to sign and verify resolutions.

Reviewer: ```suggestion
We recommend using [GnuPG](https://gnupg.org) or [Keybase](https://keybase.io) command line tools, to sign and verify signatures.
Alternatively, you may verify easily through the Keybase web interface (see below).
```

Reviewer: Sorry, was slow on this one. heh nbd

- Signing with **GnuPG**:

    ```
    $ gpg --sign --armor \
      --output resolution-001.md.github_handle.asc \
      resolution-001.md
    ```

    with **Keybase**:

    ```
    $ keybase pgp sign \
      --infile resolution-001.md \
      --outfile resolution-001.md.github_handle.asc
    ```
- Verifying with **GnuPG**:

    ```
    $ gpg --verify resolution-001.md.github_handle.asc
    ```

    with **Keybase**:

    ```
    $ keybase pgp verify \
      --infile resolution-001.md.github_handle.asc
    ```

The text of the signed document is embedded in the signature file, so you can drop any signature file `resolution-XXX.md.github_handle.asc` into [keybase.io/verify](https://keybase.io/verify) to verify a signature.
For example, you can try verifying [resolution-001.md.benhylau.asc](https://github.com/hyphacoop/organizing/blob/master/resolutions/resolution-001.md.benhylau.asc).

> 

Reviewer: ```suggestion

### Signing external documents

```

> Members also use other tools such as [HelloSign](https://hellosign.com) for signing external documents.

Reviewer: Just to not repeat ourselves and add more clarity

```suggestion
Members also use other tools such as [HelloSign](https://hellosign.com) for signing client documents.
```

Reviewer: ```suggestion
Members also use other tools such as [HelloSign](http://link.hypha.coop/inventory#HelloSign) for signing client-facing documents.
```

Could point to our own internal docs on HelloSign, which give a tiny bit more info about it (that we have a shared account, that it's currently experimental, etc.)
